net/keepalived/files/keepalived.init

```diff
@@ -371,7 +371,8 @@ process_config() {
 	rm -f $KEEPALIVED_CONF
 
 	# First line
-	printf "! Configuration File for keepalived (autogenerated via init script)\n\n" > $KEEPALIVED_CONF
+	printf "! Keepalived configuration file generated by uci\n" > $KEEPALIVED_CONF
```

Comment:
IMO there's greater benifit keeping the "autogenerated by init script" - it gives the user a better hint as to where the config is generated. Thoughts?

Reply:
ok i will drop this line